docs/swagger-v2.yaml

```diff
@@ -702,12 +755,17 @@ components:
         country:
           type: string
           description: 'Pickup point''s country. Example: NO'
+        openingHours:
+          type: array
+          items:
+            type: string
+          description: 'Pickup point''s opening hours. Example: Man-Søn: 1000-2000'
```

Comment:
the quote doesnt need to be escaped, or?

Reply:
It is escaped by doubling it: ''